src/js/select2/options.js

```diff
@@ -37,7 +37,8 @@ define([
     if (this.options.language == null) {
       if ($e.prop('lang')) {
         this.options.language = $e.prop('lang').toLowerCase();
-      } else if ($e.closest('[lang]').prop('lang')) {
+      } else if (typeof $.fn.select2.defaults.defaults.language !== 'string' &&
```

Comment:
This should actually be

```js
typeof Defaults.defaults.language !== 'string'`
```

Since `Defaults` is the singleton that is imported into this class, and it's the one that we assign to `$.fn.select2.defaults`. While these should be the same, we intentionally don't enforce that `$.fn.select2.defaults` must exist.

Reply:
Ok, thanks for the feedback. I'll make that change soon 👍.